assets/app/view/game/map.rb

```diff
@@ -35,7 +35,8 @@ def render
         @start_pos = [@cols.first, @rows.first]
         @layout = @game.layout
 
-        @scale = SCALE * map_zoom
+        store(:map_zoom, @map_zoom || Lib::Storage['map_zoom'] || 1, skip: true)
```

Comment:
why did you change this? don't store with the storage, it gets hard

Reply:
Well, I wasn’t sure about this, hence its own commit. I thought it’s better to store once per render than to read 7 times (iirc) from localStorage.

Reply:
well you have to revert it